### ISSUE TITLE
wireframes

### ISSUE THREAD

**Issue by afonsopacifer** [riojs-org/blog#6]
Pessoal, a @janasouza fez alguns protótipos so site e blog, criei essa issue para ela compartilhar com todos.
@riojs-org/blog

**Comment by janasouza** [comment]
E ai 
Seguem os wireframes que o Afonso pediu :smiley:
Conteúdo fixo 
![1161](https://cloud.githubusercontent.com/assets/12633031/8608740/32c4f1ae-2674-11e5-8014-df4aba566c65.jpg)
Conteúdo de todas as páginas 
![1162](https://cloud.githubusercontent.com/assets/12633031/8608747/453d94ee-2674-11e5-9268-1837dff93213.jpg)
Conteúdo post no blog
![1163](https://cloud.githubusercontent.com/assets/12633031/8608750/525ae816-2674-11e5-9140-e369810ba3dc.jpg)

Amanhã eu vou colocar as versões mobile.

**Comment by pedropolisenso** [comment]
Eu curti a estrutura e a proposta da ideia, muito boa! Vou tentar criar algo baseado nessa estrutura! Eu pensei em criar o site em HTML/jade mesmo, sei la e deixar apenas a estrutura do jekyll para o blog, o que acham?

CC: @riojs-org/blog

**Comment by pedromarins** [comment]
Excelente trabalho @janasouza! Curti bastante!

To mandando meus pitacos. Avaliem ai!
![32c4f1ae-2674-11e5-8014-df4aba566c65](https://cloud.githubusercontent.com/assets/215004/8619222/70c7baa0-26eb-11e5-8e31-ac0096e3f09e.jpg)
![453d94ee-2674-11e5-9268-1837dff93213](https://cloud.githubusercontent.com/assets/215004/8619227/7431bdb2-26eb-11e5-96c5-cbc1a324199a.jpg)
![525ae816-2674-11e5-9140-e369810ba3dc](https://cloud.githubusercontent.com/assets/215004/8619229/77299ac6-26eb-11e5-8f74-4c141150408c.jpg)

**Comment by pedropolisenso** [comment]
Bom, vamos lá!
- Sobre o texto inicial sobre acho interessante sim, mas precisamos saber se o site vai ter page ou será one page.
- Sobre as redes sociais, acho que não precisa de taaaaaanto destaque no sentido de ser Grande, colocando ela nas principais posições está ok (Header e Footer).
- Sobre mantenedores, e as fotos não terá necessidade é só colocar: Organização Riojs e o link do github.

-Timeline está show
- Sobre os projetos, a ideia do @pedromarins ébem legal, já vir do github pela sua API ;)
- A idia de ter uma página informando os colaboradores é legal, mas não vejo necessidade, para isso existe o github.
- O que o meu xará @pedromarins falou sobre o lance de Title/date/author faz sentido.

**Comment by afonsopacifer** [comment]
No caso, os colaborados são os 15 responsaveis por escrever..
Os mantenedores são os que vão manter/revisar tudo..

O objetivo de ter uma pagina com os colabores, seria motivar o cara a escrever, para ele ter o nome dele com a foto (que pode vir da api do github tbm), assim o kra pode mostrar para os amigos que é um escritor do blog riojs, o que acham?
@pedropolisenso @pedromarins 

Acho tbm que deveriamos marcar um hangout de leve para deixar isso mais claro :)

**Comment by leaofelipe** [comment]
Achei o wireframe bem maneiro @janasouza!

Minha opinião sobre a página de cargos é semelhante a do @pedromarins. Não acho que tenha essa nessecidade ou exista cargos. Acho que a página de Conbribuidores já é suficiente para informar quem está por trás dos projetos.

Acho inclusive que a página de Conbribuidores poderia ter também alguma forma de incentivar as pessoas a participarem e contribuirem.

**Comment by pedropolisenso** [comment]
A página de contribuintes poderia ser semelhante a do http://braziljs.org/projetos/ onde tem os projetos e os colaboradores envolvidos!

**Comment by afonsopacifer** [comment]
@pedropolisenso sim, no caso seria a pagina de projetos. A pagina de contribuidores seria para quem escreve no blog saka, para incentivar o cara a querer tem seu nome la.

@leaofelipe, nao tem pagina de cargos, so tem o cargo la como um dado do cara que escreve, mas a pagina nao é para isso, eu vou ver com a @janasouza, e posto aqui quais são as pagina pensadas e qual o conteudo e por que de cada uma, acho que vai deixar tudo mais claro :D

**Comment by pedropolisenso** [comment]
Sim, claro. Vamos levar em consideração que é apenas uma ideia esse wireframe, certo? vamos definindo aos poucos.

**Comment by afonsopacifer** [comment]
@pedropolisenso, isso :D

**Comment by pedromarins** [comment]
Agora entendi a ideia dos colaboradores, e gostei! Mas não podem ser só 15
né. São todas as pessoas que já colaboraram para o blog.

Em sex, 10 de jul de 2015 às 10:49, Afonso Pacifer notifications@github.com
escreveu:

**Comment by afonsopacifer** [comment]
@pedromarins concordo, mas podemos dar destaque aos autores fixos, o o que acha?

**Comment by pedromarins** [comment]
o problema de destacar o fixo, é que o fixo só é fixo até ele vacilar! :D

Mas acho válido destacar sim!

Em sex, 10 de jul de 2015 às 11:33, Afonso Pacifer notifications@github.com
escreveu:

**Comment by afonsopacifer** [comment]
hahaha, 
vacilou ralou.

**Comment by danilorb** [comment]
@janasouza , Se precisar de algum help no Ux ou Ui pode me chamar.

**Comment by janasouza** [comment]
@danilorb valeu :) 

Provavelmente ainda hoje eu vou enviar outra versão desses wireframes aí com as considerações de vocês. To vendo tudo isso com o Afonso (qualquer erro de informação a culpa é dele) rs

**Comment by danilorb** [comment]
Haha....  Mas está ficando bacana , parabéns.

**Comment by janasouza** [comment]
Olá
Então, seguem os wireframes mais arrumados. 

Index - Sobre 
![wireframe sobre](https://cloud.githubusercontent.com/assets/12633031/8634660/2d1e2bac-27d8-11e5-9736-028900e450ab.jpg)

Eventos 
![wireframe eventos](https://cloud.githubusercontent.com/assets/12633031/8634662/45a03fda-27d8-11e5-84a3-b322247f421f.jpg)

Projetos
Não colocamos as fotos dos contribuidores (como no BrazilJS) mas podemos colocar. O que vocês acham?
![wireframe projetos](https://cloud.githubusercontent.com/assets/12633031/8634674/b606c82a-27d8-11e5-953a-b83fa6480b02.jpg)

Blog
Cada página irão ter 7 posts, representando cada artigo da semana. 
![wireframe blog](https://cloud.githubusercontent.com/assets/12633031/8634677/d860178c-27d8-11e5-8f79-7c3d0d8277d2.jpg)

Blog post
Fui lá dar uma avaliada no Medium e usei as referências de lá. 
![wireframe blog post](https://cloud.githubusercontent.com/assets/12633031/8634689/40f3a750-27d9-11e5-9997-477b4be4e0e0.jpg)

Colaboradores
Depois de muitas dúvidas sobre essa página, espero que esse wireframe explique melhor :) 
Estamos pensando em uma forma de adicionar todos os autores, não só os 15 fixos. Conto com sua ajudinha @danilorb :D 
![wireframe colaboradores](https://cloud.githubusercontent.com/assets/12633031/8634695/9bf28db0-27d9-11e5-8022-c368d6e80cd4.jpg)

**Comment by pedromarins** [comment]
Gostei muito! Únicas coisas que mudaria:
- Home: tirar as fotos das pessoas, deixar um link para a página de colaboradores.
- Eventos: adicionar link para slides da palestra e link para video da palestra, se existirem
- Projetos: com certeza adicionar as fotos dos colaboradores!
- Colaboradores: não acho que deveriam ter descrição, e sim uma lista de links de artigos já publicados

De resto está tudo awesome! Vamos começar a codar isso!

**Comment by pedropolisenso** [comment]
Show eu to com o @pedromarins nessas opiniões aee, vamos começar a codar. Vou preparar a estrutura do blog para o modelo que segue. Temos que dividir a galera do Blog e do Website em times assim focamos ráppido em 2 projetos. Será que @janasouza pode fechar a arte final já? com a proposta de cores? e se precisa de algo só falar.

CC: @riojs-org/blog

**Comment by afonsopacifer** [comment]
Pessoal, junto com a @janasouza, peguei as ideias do @pedromarins  e @pedropolisenso, adicionamos ao layout e estamos terminando o mockup de todas as paginas, provavelmente amanha pela manhã postaremos aqui :), mas será um mockup para darmos uma olhada nas infos e identidade visual do projeto, ai poderemos discutir os ultimos detalhes do layout, assim que estiver tudo certinho, a @janasouza  vai passar o PSD organizado + Guia de Estilos certinho :D

@riojs-org/team-blog

**Comment by afonsopacifer** [comment]
@pedromarins, na pagina de eventos a ideia é que tenha um link no nome da talk para os slides,
Na pagina colaboradores, eu e a @janasouza  achamos super importante ter um mini descrição do cara que escreve os posts, por que se não, será só um nome away, sem falar que "agrega valor" (rsrsr) e a ideia da lista de artigos é muito boa, falei com a jana e ela disse que isso pode, com o tempo, quebrar o layout da pagina, e ela sugeriu uma pagina especifica para listar os posts do cara, alem de ter sua foto e descrição, assim como no [tableles](http://tableless.com.br/author/diegoeis/).
O que vocês acham?

@riojs-org/team-blog

**Comment by pedromarins** [comment]
Concordo com a página específica de cada colaborador! Acho que é bom para
agrupar os artigos.

Então vamos mover a descrição para a página própria, para não ficar
redundante. A página dos colaboradores ficam só o nome e as fotos, com o
link para a página completa!

E a busca? Tem wireframe dela?

**Comment by afonsopacifer** [comment]
Sobre a busca, na verdade um campo de busca não estava previsto nos primeiros wireframes, amanha vamos postar os mockups, podemos ir discutindo enquanto vemos a questão da busca :)

**Comment by janasouza** [comment]
Então gente, aqui está o mockup com a proposta de cores, espaçamento e organização do conteúdo. 
A página de blog está faltando, e a específica do autor também, pois foi decidido de última hora. 
Lembrando que assim que estiver tudo certo, eu e o @afonsopacifer vamos mandar o guia de estilos junto com o psd.

![mockup sobre](https://cloud.githubusercontent.com/assets/12633031/8725516/071ca7f4-2bab-11e5-82a8-ba1ed3bd32c7.jpg)

![mockup eventos](https://cloud.githubusercontent.com/assets/12633031/8725531/1617fb46-2bab-11e5-9c89-8ec5bf5e0d8a.jpg)

![mockup projetos](https://cloud.githubusercontent.com/assets/12633031/8725536/1ad3f392-2bab-11e5-832d-f75c56094886.jpg)

![mockup colaboradores](https://cloud.githubusercontent.com/assets/12633031/8725540/1e7f0892-2bab-11e5-8a81-df2b3a2d90e0.jpg)

**Comment by pedromarins** [comment]
Porra! Ficou irado!

Curti muito! Vamos sentar o pau nisso aí para publicar logo!

**Comment by andrevvalle** [comment]
Ficou irado, parabéns galera ;)

**Comment by raphamorim** [comment]
Boaaaaaa. Congrats!

**Comment by pedropolisenso** [comment]
@riojs-org/team-blog Pessoal estamos de acordo com o tema do blog, certo? então vamos dar um start no code da paradinha. Vem novidades para nós aee ein na issue #9

**Comment by pedromarins** [comment]
@afonsopacifer @janasouza conseguimos os psd's disso urgente? Estamos aqui no Hack Day e dai falta! :D

**Comment by pedromarins** [comment]
@afonsopacifer @janasouza podem mandar os psd's para nós? Só publicar aqui!

**Comment by afonsopacifer** [comment]
@pedromarins O PSD não entrou aqui.
Te mandei por e-mail agora :)

**Comment by felipemrodrigues** [comment]
Galera, por onde andam os psds?

**Comment by afonsopacifer** [comment]
@felfiz ,eu tenho aqui, mas de qualquer forma estamos terminando de montar o guia de estilos, assim que estiver ok, ele ficara publico para todos seguirem a guide line certinha :)

**Comment by felipemrodrigues** [comment]
Ótimo. Fico no aguardo :)
